torch/fx/experimental/proxy_tensor.py

```diff
@@ -27,6 +28,12 @@
 CURRENT_DECOMPOSITION_TABLE: Dict[torch._ops.OpOverload, Callable] = {}
 
 
+def fake_signature(fn, nargs):
+    """FX gets confused by varargs, de-confuse it"""
+    argnames = ",".join(f"arg{i}" for i in range(nargs))
+    return eval(f"lambda {argnames}: fn({argnames})", {"fn": fn})
```

Comment:
make_fx doesn't support kwargs right now?

Reply:
it doesn't. Actually @IvanYashchuk added another wrapper to make kwargs work with pytree...